coder_sniffer/Drupal/Sniffs/InfoFiles/DependencyArraySniff.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * Largely copied from
+ * PHP_CodeSniffer\Standards\Squiz\Sniffs\Classes\ClassFileNameSniff.
+ *
+ * Extended to support anonymous classes and Drupal core version.
+ */
+namespace Drupal\Sniffs\InfoFiles;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use Symfony\Component\Yaml\Yaml;
+
+/**
+ * Dependency should have single entry as an array in Drupal 8 info files.
+ *
+ * @category PHP
+ * @package  PHP_CodeSniffer
+ * @link     http://pear.php.net/package/PHP_CodeSniffer
+ */
+class DependencyArraySniff implements Sniff
```

Comment:
I think we should call this DependenciesArraySniff because the key in the info.yml file is "dependencies". Same for the test file.

Reply:
I will fix this.